f5/bigip/resource.py

```diff
@@ -932,3 +985,42 @@ def load(self, **kwargs):
         newinst = self._stamp_out_core()
         newinst._refresh(**kwargs)
         return newinst
+
+
+class Stats(PathElement):
+    """This class allows attaching 'stats' object as an
+
+        attribute to a resource.
+
+        Stats are available on some objects, and require
+        the target object to be loaded or created first.
+
+        It also allows the caller to retrieve the converted json
+        response in unprocessed state.
+    """
+    def __init__(self, container):
+        super(Stats, self).__init__(container)
+        self._update_stats(self._get_stats_raw())
+
+    @property
+    def stats_raw(self):
```

Comment:
this will be misleading, raw is a call to __dict__ of the class, stats_raw is a json blob that is used for later processing, this was added as an option so the user of the SDK will be able just to grab the json and process it as they see fit